interface/modules/zend_modules/module/Carecoordination/view/carecoordination/encountermanager/index.phtml

```diff
@@ -311,7 +311,7 @@ foreach($this->status_details as $row_status){
                                         <input class="check_pid check_pid_<?php echo $this->escapeHtml($row['pid']);?>" type="checkbox" name="ccda_pid[]" value="<?php echo $this->escapeHtml($row['pid']);?>" <?php if($this->form_data['select_all'] ==1) echo "checked"; ?>>                                        
                                 </td>
                                 <td style="padding: 5px;">
-                                        <a href="<?php echo $this->basePath();?>/encounterccdadispatch/index?combination=<?php echo $this->escapeHtml($row['pid']);?>&view=1&recipient=self&param=<?php echo $_SESSION['authId'];?>" target="_blank" style="text-decoration: none;">
+                                        <a href="<?php echo $this->basePath();?>/encounterccdadispatch/index?combination=<?php echo $this->escapeHtml($row['pid']);?>&view=1&recipient=self&param=<?php echo $_SESSION['authId'];?>" rel="noopener" target="_blank" style="text-decoration: none;">
```

Comment:
I think this one may break something. Thoughts @sjpadgett ?

Reply:
actually, I don't think so, since appears to be just viewing something

Reply:
yep, it's to view a ccda, so we are good

Reply:
Should be fine. Believe this is a ccda view.